apps/src/code-studio/pd/professional_learning_landing/LandingPageWorkshopsTable.jsx

```diff
@@ -205,13 +191,23 @@ class WorkshopsTable extends React.Component {
             <p>{workshop.location_address}</p>
           </div>
         </td>
-        {this.props.forMyPlPage && <td>{workshop.status}</td>}
+        {!this.props.participantView && <td>{workshop.status}</td>}
         <td>{this.renderWorkshopActionButtons(workshop)}</td>
       </tr>
     );
   }
 
   render() {
+    console.log(this.props);
```

Comment:
Same for this console log statement?

Reply:
ugh this one was even added today 🙃

Reply:
Hehe no worries, after reading this I checked my most recent PR and saw I had also committed console logs 🙃